packages/deprecation-crawler/src/lib/commands/history.command.ts

```diff
@@ -60,7 +60,7 @@ export const historyCommand: YargsCommandObject = {
 
       const deprecations: Deprecation[] = [];
       for (const tag of tagsToCrawl) {
-        const tasks = [checkout, crawl, addVersion, commitChanges];
+        const tasks = [checkout, crawl, addVersion, ensureCleanGit];
```

Comment:
Why are we using `ensureCleanGit` here? 
This stashes the changes, so the user would have to unstash them and then commit the changes?

Reply:
It was a way it worked for me to crawle/walkthrough  a changing file structure.

I should have use revert imho. I guess i didn't though too much. Plus I was not 100% why the walkthrough broke... 

I'll have a proper think on Saturday latest ✌️

Reply:
I use `git clean -f` now